### PR TITLE
Add manuscript edition to the bibliographic meta data

### PR DESCRIPTION
**Pull request by wrznr**
Taken from `mods:edition`, put into `tei:biblFull/editionStmt`.

### CODE REVIEW

File: mets_mods2teiHeader/api/mets.py [hunk 1]
```diff
@@ -280,6 +289,12 @@ def get_publishers(self):
         """
         return self.publishers
 
+    def get_edition(self):
```
Comment (kba) [inline mets_mods2teiHeader/api/mets.py]: What's the advantage of `self.get_edition()` over `self.edition`?

I think we had that discussion before though but I forgot your reasoning.

File: mets_mods2teiHeader/api/tei.py [hunk 2]
```diff
@@ -111,6 +111,15 @@ def add_publisher(self, publisher):
         name.text = publisher
         publication_stmt.insert(0, publisher_node)
 
+    def add_source_edition(self, manuscript_edition):
+        """
+        Adds an edition statement with details on the source manuscript.
+        """
+        bibl_full = self.tree.xpath('//tei:fileDesc/tei:sourceDesc/tei:biblFull', namespaces=ns)[0]
```
Comment (kba) [inline mets_mods2teiHeader/api/tei.py]: Will that xpath always return results? If not, an error with that xpath might be more intuitive than an IndexError.

File: mets_mods2teiHeader/api/mets.py (hunk 1 above)
Comment (wrznr) [reply mets_mods2teiHeader/api/mets.py]: It's basically a left-over from C++ times. Obviously, getter/setter are not recommended (http://dirtsimple.org/2004/12/python-is-not-java.html) in Python. #21

File: mets_mods2teiHeader/api/tei.py (hunk 2 above)
Comment (wrznr) [reply mets_mods2teiHeader/api/tei.py]: #22

**Pull request merged**